.github/workflows/publish.yml

```diff
@@ -3,17 +3,25 @@ name: Publish
 on:
   push:
     tags:
-      - 'v*'
+      - "v*"
 jobs:
-  build:
-    name: Build
+  build_and_check:
```

Comment:
We should probably name this `publish`.  It was incorrect when i called it `build`.